test/api/options-run-scripts.js

```diff
@@ -46,7 +46,7 @@ describe("API: runScripts constructor option", () => {
         virtualConsole.on("jsdomError", err => {
           try {
             assert.strictEqual(err.type, "unhandled exception");
-            assert(err.detail.stack.includes("at about:blank:2"));
+            assert.isTrue(err.detail.stack.includes("at about:blank:2"));
```

Comment:
I made this change since the error given from the failing `assert()` was a bit cryptic, and in order to be consistent with the rest of the tests.